src/server/page_server.cpp

```diff
@@ -30,13 +30,24 @@
 #include "page_buffer.h"
 #include "server_type.hpp"
 #include "system_parameter.h"
+#include "thread_entry_task.hpp"
```

Comment:
```suggestion
#include "thread_manager.hpp"
```
To use `create_worker_pool ()`, thread_manager.hpp is required. Maybe it is already declared in some headers

Reply:
done.